pinot-plugins/pinot-input-format/pinot-parquet/src/main/java/org/apache/pinot/plugin/inputformat/parquet/ParquetNativeRecordExtractor.java

```diff
@@ -209,79 +207,26 @@ public static long convertInt96ToLong(byte[] int96Bytes) {
   }
 
   public Object[] extractList(Group group) {
```

Comment:
There is a bug in parquet extractor where we don't call extractList if the parquet schema doesn't have ListType logical annotation

in such cases what I've found is that instead of getting output as `[1,2,3,4]` we get `[{"element": 1}, {"element": 2}, ..]

maybe we can fix that in this PR as well

Reply:
Cool, i'll create a seperate PR to fix that issue then